### PR TITLE
ci: do not report coverage on pull requests

### PR DESCRIPTION
**Pull request by nashif**
The code coverage reports on pull requests has been inconsistent and
confusing. Disable them for pull-requests.

### CODE REVIEW

**Comment by aescolar** [comment]
Oh well..
On the plus side, it will save 30 seconds of CI time per PR

**Comment by nashif** [comment]
> On the plus side, it will save 30 seconds of CI time per PR

10 minutes actually...

**Review by galak — APPROVED** [review]

File: scripts/ci/run_ci.sh [hunk 1]
```diff
@@ -122,33 +122,38 @@ function handle_coverage() {
...
-		--remove lcov.pre.info mylib.c \
-		--remove lcov.pre.info tests/\* \
-		--remove lcov.pre.info samples/\* \
-		--remove lcov.pre.info ext/\* \
-		--remove lcov.pre.info *generated* \
-		-o lcov.info --rc lcov_branch_coverage=1;
-
-	# Cleanup
-	rm lcov.pre.info;
-	rm -rf sanity-out out-2nd-pass;
```
Comment (aescolar) [inline scripts/ci/run_ci.sh]: this line should be below the if

**Review by aescolar — CHANGES_REQUESTED** [review]

File: scripts/ci/run_ci.sh (hunk 1 above)
Comment (nashif) [reply scripts/ci/run_ci.sh]: copied it after the conditional to make sure it happens in any case.

**Review by marc-hb — COMMENTED** [review]
I think it'd be useful to mention https://github.com/codecov/codecov-bash/issues/83 in the commit message, it is the main reason why I think this check is useless. Are there others?

**Review by aescolar — APPROVED** [review]
If it must be done..

**Pull request merged**